### PR TITLE
Fixed formatting for the list of OneDrives

### PR DESCRIPTION
**Pull request by YuriySamorodov**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/list-onedrive-urls.md [hunk 1]
```diff
@@ -79,12 +79,14 @@ The list you create in these steps will be saved to a text file.
 
 After the script successfully completes, a text file is created in the location specified by the **$LogFile** variable in the script. This file contains a list of all OneDrive Urls in your organization. The following text provides an example of how the list of Urls in this file should be formatted.
```
Comment (JohanFreelancer9) [inline OneDrive/list-onedrive-urls.md:80]:
```suggestion
After the script successfully completes, a text file is created in the location specified by the **$LogFile** variable in the script. This file contains a list of all OneDrive URLs in your organization. The following text provides an example of how the list of URLs in this file should be formatted.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @YuriySamorodov, not related to this PR, but kindly commit the additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by khayam123670** [comment]
Pull requests can only be merged by people who have write access to the [repository.](https://reviewscrush.com/)

**Comment by YuriySamorodov** [comment]
Hi @scanum , could you please resolve conflicts here? Looks like I have to permission to do that.

**Pull request merged**